third-party-synthetic/active-gate-extensions/extension-third-party-dns/src/dns_extension.py

```diff
@@ -29,7 +29,15 @@ def query(self, **kwargs) -> None:
             success, response_time = test_dns(dns_server, host)
             log.info(f"DNS test, DNS server: {dns_server}, host: {host}, success: {success}, time: {response_time} ")
 
-            self.client.report_simple_test(name, location, success, response_time, test_type="DNS", interval=frequency * 60)
+            self.client.report_simple_test(
+                name,
+                location,
+                success,
+                response_time,
+                test_type="DNS",
+                interval=frequency * 60,
+                edit_link=f"#settings;gf=all/customextension;id={self.plugin_info.name};gf=all",
```

Comment:
Both of the `;gf=all`s are meaningless I believe.

Reply:
Removed the `;gf=all` portion, and confirmed the links work without it